warcit/warcit.py

```diff
@@ -271,6 +487,24 @@ def make_record(self, writer, file_info):
             self.logger.debug('Writing "{0}" ({1}) @ "{2}" from "{3}"'.format(url, warc_content_type, warc_date,
                                                                               file_info.full_filename))
 
+            
+        self.write_logfile({
```

Comment:
@despens I would suggest outputting either CDXJ or JSON lines with custom fields instead of CSV, to be consistent with formats we used by warcio and pywb, or is there a particular reason for using CSV? I guess could make JSON log output a separate option.

Reply:
OK, if CSV is useful, then lets stick with that :) Was just thinking the json could more easily be used for further processing, but if its mostly for manual then perhaps csv is best.